README.md

```diff
@@ -150,6 +146,21 @@ packager URL directly, first add a Chrome extension to send an
 `AMP-Cache-Transform: any` request header. Otherwise, follow the above
 "Demonstrate privacy-preserving prefetch" instructions.
 
+##### Security Considerations
+
+Signed exchanges come with some [security
+considerations](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#security-considerations)
+that publishers should consider. A starting list of recommendations based on
+that:
+
+ * Use different keys for the signed exchange cert and the TLS cert.
+ * Only sign public content that's OK to be shared with crawlers.
+ * Don't sign personalized content. (It's OK to sign content that includes
+   static JS that adds personalization at runtime.)
+ * Be careful when signing inline JS; if it includes a vulnerability, it may be
```

Comment:
Vulnerabilities exist outside of JS, but I was trying to provide slightly more concrete advice than the spec, based on my belief of where they're the most prevalent. Perhaps there's an option that's both general and concrete? Or perhaps just stick with abstract in this case?

Reply:
I think it's OK. The part I don't like is "Be careful" isn't very actionable, not that you are trying to have a specific opinion beyond the discussion in the draft.